en/micro-integrator/docs/administer-and-observe/working-with-management-api.md

```diff
@@ -748,3 +748,69 @@ The management API has multiple resources to provide information regarding the d
 	```bash
 	curl -X GET "https://localhost:9164/management/data-services?dataServiceName=StudentDataService" -H "accept:          application/json" -H "Authorization: Bearer TOKEN" -k -i
 	```
+### GET LOG LEVEL
+
+-	**Resource**: `/logging?loggerName={logger}`
+
+    **Description**: Retrieves information related to a specific logger.
+    
+    **Example**:
+    
+    ```bash tab='Request'
+    curl -X GET "https://localhost:9164/management/logging?loggerName=org-apache-coyote" -H "accept: application/json" -H "Authorization: Bearer Token" -k
+    ```
+    ```bash tab='Response'
+    {
+      "loggerName": "org-apache-coyote",
+      "level":"WARN",
+      "componentName":"org.apache.coyote"
+    }
+    ```
+
+### UPDATE ROOT LOG LEVEL
+
+-	**Resource**: `/logging`
+
+    **Description**: Updates the log level of root logger.
+    
+    **Example**:
+    
+    ```bash tab='Request'
+    curl -X PATCH \
```

Comment:
@Vathsan This only works with PATCH method?

Reply:
Yes, we need to send a PATCH request